approval.go

```diff
@@ -97,6 +98,32 @@ Respond %s to continue workflow or %s to cancel.`,
 	return nil
 }
 
+func (a *approvalEnvironment) saveOutputs(outputs map[string]string) (bool, error) {
```

Comment:
sorry for the double tap - I actually just had a PR merged that also has a saved outputs function, but it's not for multiple outputs like this one - would you be willing to rebase and we can combine them into one function that's accessible/used here and in main.go? looks great!

Reply:
and will require a rebase off main as well

Reply:
Sure! Feel free to make any changes you believe are necessary. I'm lacking the time to dedicate to this issue right now. You can even fork it and assume as the author. This PR can even be closed in favor of yours.

Reply:
Sure! thanks